test/basic-querying.test.js

```diff
@@ -129,6 +129,16 @@ describe('basic-querying', function() {
 
     before(seed);
 
+    before(function setupDelayingLoadedHook(cb) {
```

Comment:
Nitpick: I would prefer adding a description of what you are doing here as opposed to have it as a function name:
```
before('setup delaying loaded hook', function(done) {
...
```

Feel free to ignore it.